go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/ethereum/evmc
+
+go 1.13
+
+require github.com/ethereum/go-ethereum v1.9.9
```

Comment:
Is this needed for testing? Does it mean that anyone using evmc as a god module also will depend on geth?

Reply:
I believe so. We are only using `github.com/ethereum/go-ethereum/common` but go-ethereum only defines single module for whole source repo. Maybe there is a way to define the `github.com/ethereum/go-ethereum/common` module there?

Reply:
I'm also not sure if go-ethereum will be able to use evmc module with this.

Reply:
This is solved by #499.